experimentation/src/test/java/com/automattic/android/experimentation/local/FileBasedCacheTest.kt

```diff
@@ -0,0 +1,68 @@
+package com.automattic.android.experimentation.local
+
+import com.automattic.android.experimentation.domain.Assignments
+import com.automattic.android.experimentation.domain.Variation.Control
+import com.automattic.android.experimentation.domain.Variation.Treatment
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.junit.Test
+import kotlin.io.path.createTempDirectory
+
+@OptIn(ExperimentalCoroutinesApi::class)
+internal class FileBasedCacheTest {
+
+    private lateinit var sut: FileBasedCache
```

Comment:
Minor (🔍): Consider deleting this `lateinit` field and instead directly using a `val` variable per test. I think doing that will make it much more straightforward that this is exactly what each test needs and removing this extra `field` related boilerplate.

- From: `sut = fileBasedCache()`
- To: `val sut = fileBasedCache()`

Reply:
Good point, thanks!

Reply:
Addressed in 4f8927f